<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>751</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>대시 보드</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>330</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>현재 매장 내 과일</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>431</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>현재 매장 내 인원</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="userGroupBox1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>User1</string>
     </property>
     <widget class="QLabel" name="userNameLabel1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>사용자 이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="userCart1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>311</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;사과 0개&lt;/p&gt;&lt;p&gt;복숭아 0개&lt;/p&gt;&lt;p&gt;체리 0개&lt;/p&gt;&lt;p&gt;딸기 0개&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="userGroupBox1_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>User2</string>
     </property>
     <widget class="QLabel" name="userNameLabel2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>사용자 이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="userCart2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>311</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;사과 0개&lt;/p&gt;&lt;p&gt;복숭아 0개&lt;/p&gt;&lt;p&gt;체리 0개&lt;/p&gt;&lt;p&gt;딸기 0개&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="userGroupBox3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>User3</string>
     </property>
     <widget class="QLabel" name="userNameLabel3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>사용자 이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="userCart3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>311</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;사과 0개&lt;/p&gt;&lt;p&gt;복숭아 0개&lt;/p&gt;&lt;p&gt;체리 0개&lt;/p&gt;&lt;p&gt;딸기 0개&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="userGroupBox1_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>380</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>User4</string>
     </property>
     <widget class="QLabel" name="userNameLabel4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>사용자 이름</string>
      </property>
     </widget>
     <widget class="QLabel" name="userCart4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>311</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;사과 0개&lt;/p&gt;&lt;p&gt;복숭아 0개&lt;/p&gt;&lt;p&gt;체리 0개&lt;/p&gt;&lt;p&gt;딸기 0개&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>550</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>매장 총 매출 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="sellingsum">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>600</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>900000원</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="shelves_table">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>380</y>
       <width>341</width>
       <height>161</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>품종</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>상한 개수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>정상 개수</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="CCTV_1">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>100</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Logs</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>79</y>
       <width>1071</width>
       <height>621</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Sell</string>
      </attribute>
      <widget class="QTableWidget" name="selling_table">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>511</width>
         <height>541</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>품종</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>판매 갯수</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>판매 가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>판매 시간</string>
        </property>
       </column>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>241</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>누적 매출액</string>
        </property>
       </widget>
       <widget class="QLabel" name="sellingsum2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>8000 원</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>241</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>매대의 과일</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>10 개</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>300</y>
         <width>241</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>상한 과일 개수</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>10 개</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>440</y>
         <width>241</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>판매 과일 개수</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>10 개</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Visitor</string>
      </attribute>
      <widget class="QTableWidget" name="visit_table">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>20</y>
         <width>411</width>
         <height>541</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>분류</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>신원</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>시간</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>30</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yy/M/d/</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2024</year>
       <month>12</month>
       <day>6</day>
      </date>
     </property>
    </widget>
    <widget class="QPushButton" name="seeAllDaybtn">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>See All Day</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Theft</string>
    </attribute>
    <widget class="QTableWidget" name="event_table">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>511</width>
       <height>651</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>분류</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>신원</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>video</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>시간</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>CCTV</string>
    </attribute>
    <widget class="QLabel" name="CCTV_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>1041</width>
       <height>651</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
